play/src/components/GameSessionSwitch.tsx

```diff
@@ -50,13 +50,20 @@ export default function GameSessionSwitch({
     (isShortAnswerEnabled
       ? currentQuestion?.responses?.reduce(
           (acc: IChoice[], response: IResponse) => {
-            console.log(response);
-
+            const shouldAddResponse = 
```

Comment:
In Phase Two, we only want to display answers that the teacher has selected via Featured Mistakes or the correct answer. This accomplishes that.